src/picongpu/include/particles/creation/creation.kernel

```diff
@@ -0,0 +1,307 @@
+/**
+ * Copyright 2015-2016 Heiko Burau, Macro Garten
```

Comment:
Btw, how are the licensing conventions for the order of names when you just relocate and rename code?
In principle this is nearly exactly my `ionization.hpp` with a few renamings but no structural changes. All the other persons that were credited in my file are missing here, now.